docker/service.go

```diff
@@ -103,12 +113,11 @@ func (s *Service) constructContainers(create bool, count int) ([]*Container, err
 			dockerContainer, err := c.Create(imageName)
 			if err != nil {
 				return nil, err
-			} else {
-				logrus.Debugf("Created container %s: %v", dockerContainer.Id, dockerContainer.Names)
 			}
+			logrus.Debugf("Created container %s: %v", dockerContainer.Id, dockerContainer.Names)
 		}
 
-		result = append(result, NewContainer(client, containerName, s))
+		result = append(result, c)
```

Comment:
I felt like this was a duplicate from https://github.com/docker/libcompose/pull/39/files#diff-9369bcb8195200a9e0e57e4fe7dda0b2L95 : we just call `NewContainer` with the same parameter, so :stuck_out_tongue:.